src/navigation/FavoritesNavigator.js

```diff
@@ -28,14 +31,22 @@ export function FavoritesNavigator() {
 }
 
 function FavoritesHeader() {
+  const navigation = useNavigation();
+
   return (
     <View style={styles.container}>
-      <View style={styles.containerIcon}>
-        <Image
-          style={styles.iconHeader}
-          source={backIcon}
+      <View>
+        <TouchableWithoutFeedback
+          accessibilityRole="button"
+          onPress={() => navigation.jumpTo(TABS.home)}
```

Comment:
Avoid JSX on arrow functions. Use `useCallback` hook.
```suggestion
          onPress={jumpToHomeTab}
```